.changeset/flat-chefs-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Truncate the entity name on the system diagram
```

Comment:
Truncate long entity names on the system diagram

Reply:
Updated!